### ISSUE TITLE
Fix ratelimit

### ISSUE THREAD

**Issue by Buried-In-Code** [Metron-Project/Simyan#178]
Currently Simyan ratelimits 20 req/min for any resource. Comicvine requires 200 req/resource/hour.

**Comment by Buried-In-Code** [comment]
Revisiting this issue, looks like they updated it to be just requiring a 1sec delay between requests. Which is alot easier to implement then a per resource limit.

https://comicvine.gamespot.com/forums/api-developers-2334/api-rate-limiting-1746419/

**Comment by xawoger** [comment]
No, their implementation according to my understanding is 200 requests per hour per resource with at least 1s between each request. Unless I am not aware of any other change that happened.

**Comment by Buried-In-Code** [comment]
Good catch, hadn't known about the 1s delay between requests
